engine/test/mock_app/mock_app_api_get_list.py

```diff
@@ -1,12 +1,14 @@
 """
 NO TITLE HERE, Title in __api__
+----------------
+Description of Test app api list
 """
```

Comment:
What about the case where there's only a description here with no delimiters and title is on event_api? Is it covered?